components/CommentModal/CommentModal.js

```diff
@@ -1,77 +1,9 @@
 import { useState } from "react";
 import styled from "styled-components";
 import useLocalStorageState from "use-local-storage-state";
-import LetterCount from "../LetterCount/LetterCount";
+import NotesInput from "../NotesInput/NotesInput";
 import { uid } from "uid";
 
-const StyledSection = styled.section`
-  display: flex;
-`;
-
-const CommentButton = styled.button`
-  width: 25px;
-  height: 25px;
-  margin-right: 10px;
-`;
-
-const ButtonWrapper = styled.section`
-  display: flex;
-  flex-direction: column;
-  width: 50%;
-  margin: 10%;
-`;
-
-const ConfirmationModal = styled.section`
-  display: flex;
-  align-items: center;
-  flex-direction: column;
-  justify-content: center;
-  background-color: rgb(255, 255, 255);
-  border-radius: 8px;
-  padding: 20px;
-  border: 3px rgb(255, 0, 0) solid;
-`;
-
-const Overlay = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
-  height: 100%;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  position: fixed;
-  background-color: rgba(49, 49, 49, 0.8);
-`;
-
-const CommentForm = styled.form`
-  display: flex;
-  align-items: center;
-  flex-direction: column;
-  justify-content: center;
-  background-color: rgb(255, 245, 238);
-  border-radius: 8px;
-  padding: 20px;
-`;
-const CommentsList = styled.section`
-  margin-top: 20px;
-  list-style: none;
-`;
-
-const StyledComment = styled.p`
-  border: 1px solid rgb(0, 0, 0);
-  background-color: rgb(255, 245, 238);
-  border-radius: 4px;
-  padding: 3px;
-  word-wrap: break-word;
-`;
-
-const StyledButton = styled.button`
-  margin: 3px;
-`;
-
 export default function CommentModal({ id }) {
   const [modal, setModal] = useState(false);
   const [skip, setSkip] = useState(false);
```

Comment:
To make the code more consistent, I would like to suggest renaming this state according to the button functionality - to cancel something 🙂
```suggestion
  const [cancel, setCancel] = useState(false);
```


💡 You could also think about  to combine these two states in a single state object, especially if both states are part of the same logic or the same UI element (in this case a modal). This can simplify the code and make it clear that these states belong together. 

Example of the concept:
```js
const [modalState, setModalState] = useState({ isOpen: false, isCancelled: false });

// to open the modal
const openModal = () => setModalState({ isOpen: true, isCancelled: false });

// to close the modal without canceling
const closeModal = () => setModalState({ isOpen: false, isCancelled: false });

// to cancel the entry
const cancelModal = () => setModalState({ isOpen: false, isCancelled: true });


```

Reply:
good idea! ✓